### PR TITLE
update README, quick test

### PR DESCRIPTION
**Pull request by davewatters**
Just added a README.md. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+DEfault README file with a typo to be fixed!
```
Comment (hashim222) [inline README.md:1]: # typo😅

**Review by hashim222 — CHANGES_REQUESTED** [review]
hi dave PR reviwed, one change needs making.

File: README.md (hunk 1 above)
Comment (davewatters) [reply README.md]: Resolved typo.

**Review by hashim222 — APPROVED** [review]
LGTM

**Pull request merged**